src/main/webapp/provenance-popups-fragment.xhtml

```diff
@@ -48,19 +49,11 @@
                                           fileUploadListener="#{provPopupFragmentBean.handleFileUpload}" 
                                           update="provPopupOutput,assignMessages" 
                                           label="#{bundle['file.editProvenanceDialog.selectToAddBtn']}"
+                                          onstart="javascript:uploadWidgetDropRemoveMsg();uploadStarted();"
+                                          onerror="javascript:uploadFailure();"
                                           oncomplete="bind_bsui_components();PF('provUpload').hide();"
                                           fileLimit="1" 
                                           invalidSizeMessage="#{bundle['file.edit.error.file_exceeds_limit']}"/>
-                            <script>
-                                function uploadStarted() {
```

Comment:
FWIW - this uploadStarted() method was overwriting the one in editFilesFragment.xhtml...

Reply:
@matthew-a-dunlap Could you confirm that this method was not needed, for the whole provenance upload thing to work? (looks like not - I'm not even seeing a button with the id "provUploadStarted"... but wanted to double-check)

Reply:
@landreev I am pretty sure it doesn't, I'm hoping to check in on it while waiting for my flight tomorrow.